change/@microsoft-teams-js-19c8476a-7a54-4481-b8cb-811fcd3bab65.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
because we are marking thecurrent `send` functions as deprecated I believe this should be a `minor` rather than a `patch`

Reply:
I marked this as patch because these functions are internal to the library and shouldn't affect library consumers at all. Although, come to think of it this may need to be upgraded to minor if we keep the changes to the authCEA capability, so I'll leave this open until we decide on that